test/worlds/imu_named_frame.sdf

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
+  <world name="imu_sensor">
+    <gravity>0 0 -5</gravity>
+    <physics name="1ms" type="ode">
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>0</real_time_factor>
+    </physics>
+    <plugin
+      filename="ignition-gazebo-physics-system"
+      name="ignition::gazebo::systems::Physics">
+    </plugin>
+    <plugin
+      filename="ignition-gazebo-imu-system"
+      name="ignition::gazebo::systems::Imu">
+    </plugin>
+
+    <spherical_coordinates>
+      <heading_deg>0</heading_deg>
+    </spherical_coordinates>
```

Comment:
Should we remove this element to exercise the alternate code-path where we're checking
`if (world.SphericalCoordinates(_ecm))` ?

Reply:
Done